storage/src/meta/mod.rs

```diff
@@ -861,7 +861,7 @@ mod tests {
             state: Arc::new(state),
         };
 
-        let vec = info.get_chunks_uncompressed(0x0, 0x1001).unwrap();
+        let vec = info.get_chunks_uncompressed(0x0, 0x1001, 1).unwrap();
```

Comment:
Use "0" instead of "1", which means batch operation is disabled?

Reply:
done.